test/cmdlineTests.sh

```diff
@@ -354,13 +354,20 @@ function test_via_ir_equivalence()
         sed '/^Binary representation:$/d'
     )
     bin_output_via_ir=$(
+        echo "$solidity_code" |
+        msg_on_error --no-stderr "$SOLC" - --via-ir --bin "${optimizer_flags[@]}" |
+        sed '/^======= <stdin>/d' |
+        sed '/^Binary:$/d'
+    )
+    bin_output_experimental_via_ir=$(
         echo "$solidity_code" |
         msg_on_error --no-stderr "$SOLC" - --experimental-via-ir --bin "${optimizer_flags[@]}" |
         sed '/^======= <stdin>/d' |
         sed '/^Binary:$/d'
     )
 
     diff_values "$bin_output_two_stage" "$bin_output_via_ir" --ignore-space-change --ignore-blank-lines
+    diff_values "$bin_output_two_stage" "$bin_output_experimental_via_ir" --ignore-space-change --ignore-blank-lines
```

Comment:
Not sure if it's worth keeping this. We're compiling some external contracts here so it's slow (this part of the cmdline tests adds 30-60 secs which is a significant part of the whole runtime). Maybe some lighter test would suffice?